commit-message.md

```diff
@@ -0,0 +1,36 @@
+# Guidelines to create great commit messages
+
+
+We use [GitCop](https://gitcop.com) to check that commit messages are
+properly written. The rules are the following:
+
+* The first line of a commit message, called the subject line should
+  not be more than 80 characters long.
+
+* The commit message should end with the following trailers:
+
+  ```
+  Licence: MIT
+  Signed-off-by: User Name <email@address>
+  ```
+
+  where "User Name" is the author's real name and email@address one of
```

Comment:
On Sat, Jun 06, 2015 at 03:19:37PM -0700, rht wrote:
> > +  where "User Name" is the author's real name and email@address one of
> 
> What is "real name"? What if the author is a vermin without a legal
> name?

Presumably it's how you'd identify yourself to a court considering a
copyright- or licence-infringement case.  I'm not sure how that breaks
down in edge cases, but I imagine most contributors will have a
government-issued ID that has a name on it, and that would probably be
a good choice ;).